yt/data_objects/static_output.py

```diff
@@ -929,9 +929,10 @@ def _find_extremum(self, field, ext, source=None, to_array=True):
         if to_array:
             if any(x.units.is_dimensionless for x in coords):
                 mylog.warning(
-                    f"dataset {self} has angular coordinates. "
+                    "dataset `%s` has angular coordinates. "
                     "Use 'to_array=False' to preserve "
-                    "dimensionality in each coordinate."
+                    "dimensionality in each coordinate.",
+                    str(self),
```

Comment:
Why `str(self)` and not `self`?

Reply:
it's equivalent in this context, but IMO it's a bit clearer to perform the conversion explicitly. Could be convinced otherwise though.

Reply:
Fair enough, I don't think this matters.